.github/workflows/docs.yaml

```diff
@@ -29,8 +29,10 @@ jobs:
           NODE_OPTIONS: --max-old-space-size=6144
 
       - name: Deploy Documentation 🚀
-        uses: JamesIves/github-pages-deploy-action@releases/v3
+        uses: JamesIves/github-pages-deploy-action@releases/v4
         with:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
we don't need this anymore as the action uses the default authentication since v4.